api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java

```diff
@@ -881,7 +881,7 @@ private boolean userHasEncounterPrivilege(Privilege privilege, User user) {
 	public void checkIfEncounterTypesAreLocked() {
 		String locked = Context.getAdministrationService().getGlobalProperty(
 		    OpenmrsConstants.GLOBAL_PROPERTY_ENCOUNTER_TYPES_LOCKED, "false");
-		if ("true".equals(locked.toLowerCase())) {
```

Comment:
Edit this and squash. Did you get a chance to look at https://wiki.openmrs.org/display/docs/Pull+Request+Tips